src/sonic-yang-models/doc/Configuration.md

```diff
@@ -1440,6 +1441,31 @@ channel name as object key, and tagging mode as attributes.
 }
 ```
 
+### VXLAN
+
+VXLAN_TUNNEL holds the VTEP source ip configuration.  
+VXLAN_TUNNEL_MAP holds the vlan to vni and vni to vlan mapping configuration.
+
+```
+{
+"VXLAN_TUNNEL": {
+        "vtep1": {
+            "src_ip": "10.10.10.10"
+        }
+  }
+"VXLAN_TUNNEL_MAP" : {
+        "vtep1|map_1000_Vlan100": {
+           "vni": "1000",
+           "vlan": "100"
+         },
+        "vtep1|testmap": {
+           "vni": "22000",
+           "vlan": "70"
+         },
+  }
+}
+```

```

Comment:
Can you please add VXLAN_EVPN_NVO details and example?

Reply:
Done